cartography/intel/aws/ec2/auto_scaling_groups.py

```diff
@@ -1,243 +1,205 @@
-        MERGE (subnet)<-[r:VPC_IDENTIFIER]-(group)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-    """
-
-    ingest_instance = """
-    UNWIND $instances_list as i
-        MERGE (instance:Instance:EC2Instance{id: i.InstanceId})
-        ON CREATE SET instance.firstseen = timestamp()
-        SET instance.lastupdated = $update_tag, instance.region=$Region
-        WITH instance
-        MATCH (group:AutoScalingGroup{arn: $GROUPARN})
-        MERGE (instance)-[r:MEMBER_AUTO_SCALE_GROUP]->(group)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-        WITH instance
-        MATCH (aa:AWSAccount{id: $AWS_ACCOUNT_ID})
-        MERGE (aa)-[r:RESOURCE]->(instance)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-    """
-
-    ingest_lts = """
-    UNWIND $autoscaling_groups_list as ag
-        MATCH (group:AutoScalingGroup{arn: ag.AutoScalingGroupARN})
-        MATCH (template:LaunchTemplate{id: ag.LaunchTemplate.LaunchTemplateId})
-        MERGE (group)-[r:HAS_LAUNCH_TEMPLATE]->(template)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-    """
-
-    ingest_lcs = """
-    UNWIND $autoscaling_groups_list as ag
-        MATCH (group:AutoScalingGroup{arn: ag.AutoScalingGroupARN})
-        MATCH (config:LaunchConfiguration{name: ag.LaunchConfigurationName})
-        MERGE (group)-[r:HAS_LAUNCH_CONFIG]->(config)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-    """
-
-    launch_configs = []
-    launch_templates = []
-    for group in data:
-        if group.get('LaunchConfigurationName'):
-            launch_configs.append(group)
-        if group.get('LaunchTemplate'):
-            launch_templates.append(group)
-
-        group['CreatedTime'] = str(group['CreatedTime'])
-
-    neo4j_session.run(
-        ingest_group,
-        autoscaling_groups_list=data,
-        AWS_ACCOUNT_ID=current_aws_account_id,
+    load(
+        neo4j_session,
+        AutoScalingGroupSchema(),
+        data,
         Region=region,
-        update_tag=update_tag,
+        AWS_ID=current_aws_account_id,
+        lastupdated=update_tag,
     )
-    neo4j_session.run(
-        ingest_lcs,
-        autoscaling_groups_list=launch_configs,
-        AWS_ACCOUNT_ID=current_aws_account_id,
+
+
+def load_vpcs(
+        neo4j_session: neo4j.Session, data: list[dict], region: str, current_aws_account_id: str, update_tag: int,
+) -> None:
+    load(
+        neo4j_session,
+        EC2SubnetAutoScalingGroupSchema(),
+        data,
         Region=region,
-        update_tag=update_tag,
+        AWS_ID=current_aws_account_id,
+        lastupdated=update_tag,
     )
-    neo4j_session.run(
-        ingest_lts,
-        autoscaling_groups_list=launch_templates,
-        AWS_ACCOUNT_ID=current_aws_account_id,
+
+
+def load_instances(
```

Comment:
```suggestion
def load_instance_asgs(
```

Or similar. Just something to indicate that this is an instance as known by an ASG